modules/cluster-logging-rn-5.5.6.adoc

```diff
@@ -0,0 +1,49 @@
+
+* Before this update the Operator would install the console view plugin even when not enabled on the cluster, causing the Operator to crash. With this update, accounts for clusters that do not have the console enabled no longer crash the Operator. (link:https://issues.redhat.com/browse/LOG-3407[LOG-3407])
+
+* Before this update, a prior fix to support a regression where the status of the Elasticsearch deployment was not being updated caused the Operator to crash unless the `Red Hat Elasticsearch Operator` was deployed. With this update, that fix has been reverted so the Operator is now stable but re-introduces the previous issue related to the reported status. (link:https://issues.redhat.com/browse/LOG-3428[LOG-3428])
+
+* Before this update, when the Loki Operator deployed with a size of `1x.small` only one pod deployed for the LokiStack gateway. With this update, two pods deploy for the gateway. (link:https://issues.redhat.com/browse/LOG-3478[LOG-3478])
+
+* Before this update, records written to Elasticsearch would fail if multiple label keys had the same prefix and some keys included included dots. With this update, underscores replace dots in label keys, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3341[LOG-3341])
+
+* Before this update, the logging view plugin contained an incompatible feature for certain versions of {product-title}. With this update, the correct release stream of the plugin resolves the issue. (link:https://issues.redhat.com/browse/LOG-3467[LOG-3467])
+
+* Before this update, the reconciliation of the `ClusterLogForwarder` custom resource would incorrectly report a degraded status of one or more pipelines causing the collector pods to restart every 8-10 seconds. With this update, reconciliation of the `ClusterLogForwarder` custom resource processes correctly, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3469[LOG-3469])
+
+* Before this change the spec for the `outputDefaults` field of the ClusterLogForwarder custom resource would apply the settings to every declared Elasticsearch output type. This change corrects the behavior to match the enhancement specification where the setting specifically applies to the default managed Elasticsearch store. (link:https://issues.redhat.com/browse/LOG-3342[LOG-3342])
+
+* Before this update, the must-gather script did not complete because oc needs a folder with write permission to build its cache. With this update, oc has write permissions to a folder, and the must-gather script completes successfully. (link:https://issues.redhat.com/browse/LOG-3472[LOG-3472])
```

Comment:
```suggestion
* Before this update, the must-gather script did not complete because `oc` needs a folder with write permission to build its cache. With this update, `oc` has write permissions to a folder, and the must-gather script completes successfully. (link:https://issues.redhat.com/browse/LOG-3472[LOG-3472])
```

Reply:
I don't think this is correct, because oc isn't an API object. After checking the OCP release notes, I think the appropriate form would be this: 

* Before this update, the the OpenShift CLI (oc) `must-gather` script did not complete because the OpenShift CLI (oc) needs a folder with write permission to build its cache. With this update, the OpenShift CLI (oc) has write permissions to a folder, and the `must-gather` script completes successfully. (link:https://issues.redhat.com/browse/LOG-3472[LOG-3472])

Reply:
Hi Libby,

Identifying the first instance of `oc` as the OpenShift CLI looks good. I'm not sure you need to do that again in the subsequent instances.

For formatting, this suggestion is based on this section of the IBM Style Guide:

"**Commands, parameters, and options in general text**
In general text, use a bold monospaced font for a command." 

And based on common practice in the openshift-doc repo. There are many examples like this one in the doc:

`* You have access to the OpenShift CLI (`oc`) that matches your {product-title} version.`

There are about 900 instances of `oc` in the doc:
[results-for-oc-in-openshift-doc.txt](https://github.com/openshift/openshift-docs/files/10534609/results-for-oc-in-openshift-doc.txt)